### PR TITLE
Sweet Survey 

### PR DESCRIPTION
**Pull request by lilihorvath**
Link to my deployed project:
sweet-survey.netlify.app
My reflections on how this week's project turned out:
OMG, I have still problems about Summary parts, but I loved the materials and the way how we studied react!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+Find bugs, when it seems everything is fine - that is a big challange in react especially when I am using different, several components. 
+- How to use form fields with React
+- How to use `useState`
+- How to use state to show different components based on where you are in the survey
 
 ## View it live
-
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+sweet-survey.netlify.app
```
Comment (AxelHammarback) [inline README.md:11]: Very sweet, Lili! Love the theme and the overall aesthetics of it :)

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,49 @@
-import React from 'react'
+import React, { useState } from 'react'
+
+import { Input } from 'Components/Input'
+import { Radio } from 'Components/Radio'
+import { Select } from 'Components/Select'
+import { Button } from 'Components/Button'
+import { Summary } from 'Components/Summary'
+
+import './index.css'
```
Comment (AxelHammarback) [inline code/src/App.js:9]: Well done on following conventions for importing (Maks will be proud)

File: code/src/App.js [hunk 3]
```diff
@@ -1,9 +1,49 @@
...
+        } 
+
+        {section === 3 &&
+          <Summary className='summary' name={name} food={food} radioInput={radioInput} />
+        }      
+
+        </form>
+    </section>
+    )       
+}
```
Comment (AxelHammarback) [inline code/src/App.js:49]: Super clear to follow along, and the (section+1) method you went with is clear, smart and intuitive.

File: code/src/index.css [hunk 4]
```diff
@@ -1,13 +1,27 @@
...
+
+.survey {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-end;
+  margin-top: 15%;
+  margin-right: 10%;
+  margin-left: 50%;
+}
```
Comment (AxelHammarback) [inline code/src/index.css:27]: Nice to see you managed to create such a distinctive style with only 27 lines of CSS, well done! Very versatile 😄

File: code/src/Components/Progress.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export const Progress = ({ current, total }) => {
+    return (
+        <p className='progress'>Sweet {current} of {total}</p>
+    )
+}
```
Comment (AxelHammarback) [inline code/src/Components/Progress.js:7]: Nice way to handle the progress, I will definitely do this in other projects 👍

File: code/src/Components/Select.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+import { Progress } from 'Components/Progress'
+
+export const Select = ({ food, setFood }) => {
+    return (
+        <form>
+            <Progress current='3' total='3'/> 
+            <h2>Here we go: select your favourite sweet from the dropdown! Oops! You can choose only one Sweetie!</h2>
```
Comment (AxelHammarback) [inline code/src/Components/Select.js:9]: More of a "UX Writing" comment, but the "Oops!" text made me think I had done something wrong when I read it first 🙀

**Review by AxelHammarback — APPROVED** [review]
Well done, Lili! Super sweet survey, I enjoyed both the site itself, as well as reading your thorough code. It was clear, consistent, and followed standards. Well done 🎉

File: code/src/App.js (hunk 2 above)
Comment (lilihorvath) [reply code/src/App.js:9]: Thank you Axel!(:

File: code/src/App.js (hunk 3 above)
Comment (lilihorvath) [reply code/src/App.js:49]: I read a lot of posts on StackOverflow...it has taken some time but definitely worth it!

File: code/src/index.css (hunk 4 above)
Comment (lilihorvath) [reply code/src/index.css:27]: Yes, that part of my project is not finished, I just have to start the next week's project - but as soon as possible I am going to fix it!

File: code/src/Components/Select.js (hunk 6 above)
Comment (lilihorvath) [reply code/src/Components/Select.js:9]: Ooo, I see your point, yes could be. In Hungary it has totally different meaning but I am going to changes it as well! Thank you Axel!

File: code/src/Components/Radio.js [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+          <input
+            type='radio'
+            value={group}
+            onChange={event => setRadioInput(event.target.value)}
+            checked={radioInput === group}
+          />
+          {group}
+        </label>
+      ))
+      }
```
Comment (HIPPIEKICK) [inline code/src/Components/Radio.js:27]: These should be a little more indented

File: code/src/Components/Select.js [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+            <select
+                onChange={event => setFood(event.target.value)}
+                value={food}
+                required
+            >
```
Comment (HIPPIEKICK) [inline code/src/Components/Select.js:14]: When we have a boolean attribute, it's convention to start with that:
```
<select
  required
  onChange={event => setFood(event.target.value)}
  value={food}
>
```

File: code/src/Components/Summary.js [hunk 9]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+
+export const Summary = (props) => {
+  const { name, radioInput, food } = props
```
Comment (HIPPIEKICK) [inline code/src/Components/Summary.js:5]: You can be consistent with how you retrieve the props:
`export const Summary = ({ name, radioInput, food }) => {`

File: code/src/Components/Radio.js (hunk 7 above)
Comment (lilihorvath) [reply code/src/Components/Radio.js:27]: Yes, you're right sorry fo that mistake!

File: code/src/Components/Select.js (hunk 8 above)
Comment (lilihorvath) [reply code/src/Components/Select.js:14]: I see, going to be fixed!

File: code/src/Components/Summary.js (hunk 9 above)
Comment (lilihorvath) [reply code/src/Components/Summary.js:5]: Ooo I see! Thank you very much for it!

**Pull request closed without merging**